tasks/scripts.mjs

```diff
@@ -41,28 +41,38 @@ export async function compileJavaScript ([modulePath, { srcPath, destPath, fileP
   const moduleSrcPath = join(srcPath, modulePath)
   const moduleDestPath = join(destPath, filePath ? filePath(parse(modulePath)) : modulePath)
 
+  // Rollup output format
+  const format = moduleDestPath.endsWith('.mjs')
+    ? 'es'
+    : 'umd'
+
   // Create Rollup bundle
   const bundle = await rollup({
-    input: moduleSrcPath
+    input: moduleSrcPath,
+
+    // Preserve JavaScript ESM import statements
+    experimentalPreserveModules: format === 'es'
```

Comment:
Regarding the "experimental" option name, there are a few unsupported Rollup features that we may need to be aware of in future with modules preserved:

1. Dynamic `import()`
2. Multiple [`input` named entries](https://rollupjs.org/configuration-options/#input) in watch mode
3. Chunk optimisation
4. Tree shaking

We don't actually need these (yet) but they arrived in later Rollup releases

I've added some background info for context:

### Rollup v0.57.0
The option `{ experimentalPreserveModules }` was added in [**Rollup v0.57.0**](https://github.com/rollup/rollup/pull/1922)

This takes us from 15 March 2018 to the last IE8 release in 28 May 2018:

```console
rollup@0.57.0
rollup@0.57.1
rollup@0.58.0
rollup@0.58.1
rollup@0.58.2
rollup@0.59.0
rollup@0.59.1
rollup@0.59.2
rollup@0.59.3
rollup@0.59.4
```

### Rollup v0.58.0
It was stabilised in [Rollup v0.58.0](https://github.com/rollup/rollup/releases/tag/v0.58.0) with some notable PRs:

**Hoisting of deep imports for performance**
(e.g. Polyfills moved into all.mjs)
https://github.com/rollup/rollup/pull/2073

**Fixes for "single file" builds like ours**
https://github.com/rollup/rollup/pull/2123

**Warnings removed for tree-shaken imports** (that were exported)
https://github.com/rollup/rollup/pull/2124

### Rollup v0.59.0

With another fix in [Rollup v0.59.0](https://github.com/rollup/rollup/releases/tag/v0.59.0)

**Fixes for file paths in source maps**
https://github.com/rollup/rollup/pull/2161

### Rollup v1.0.0
The option was renamed to `{ preserveModules }` in Rollup v1.0.0